bundles/org.eclipse.equinox.common.tests/src/org/eclipse/equinox/common/tests/AllTests.java

```diff
@@ -1,36 +0,0 @@
-/*******************************************************************************
```

Comment:
If you remove this class tests can no longer be run in I-builds as it's waht is set in test.xml https://github.com/eclipse-equinox/equinox/blob/master/bundles/org.eclipse.equinox.common.tests/test.xml#L48

Reply:
I'm a bit surprised. I assumed that the test.xml are just leftovers from the past where Maven was not used and wanted to suggest to remove them.
How are these test.xml files picked up in an I-build? AFAIK Tycho-surefire does not respect them and either runs the entire plug-in or the suite specified in corresponding properties.

Reply:
https://wiki.eclipse.org/Platform-releng/Eclipse_Test_Framework (although probably outdated in places) should explain things. https://github.com/eclipse-platform/eclipse.platform.releng/blob/master/bundles/org.eclipse.test/src/org/eclipse/test/EclipseTestRunner.java is the key part IIRC.

Reply:
Maven is not used for running the tests for I-builds as they are run on already provisioned target and this is done on 6 different os/arch/jvm combos without duplicating the whole rebuild process on each combo as maven would do.